app/templates/frameworks/updates.html

```diff
@@ -118,6 +120,7 @@
               %}
                 {% include "toolkit/forms/textbox.html" %}
               {% endwith %}
+            <p>You can also read the <a href="https://www.gov.uk/guidance/g-cloud-suppliers-guide" rel="noopener noreferrer" target="_blank">guidance on the the application process and information you must provide</p>. 
```

Comment:
The `<a href="...">` needs a closing `</a>` tag - otherwise looks good!

Reply:
Ah and the full stop should probably go inside the `<p>`.

Reply:
Thank you Kat! Have fixed :)